src/sentry/api/endpoints/organization_check_has_mobile_app_events.py

```diff
@@ -0,0 +1,74 @@
+import logging
+
+
+from django.core.cache import cache
+from datetime import timedelta
+from django.utils import timezone
+
+from sentry.api.bases import OrganizationEventsEndpointBase
+from sentry.snuba import discover
+
+# one day cache
+CACHE_TTL = 24 * 60 * 60
+
+
+logger = logging.getLogger(__name__)
+
+
+class OrganizationCheckHasMobileAppEvents(OrganizationEventsEndpointBase):
+    def get(self, request, organization):
+        # cache is unique to an org
+        cache_key = f"check_mobile_app_events.{organization.id}"
+        cache_value = cache.get(cache_key)
+        # cache miss, lookup and store value
+        if cache_value is None:
+            result = self._get(request, organization)
+            # store results in an object since we don't result=None to be a cache miss
+            cache_value = {"result": result}
```

Comment:
Do you actually need this `result` key?

Reply:
@EvanPurkhiser It's because if we get no results, I don't want `if cache_value is None` to be true. Is there a better way of setting a `None` value in the cache? I didn't want to do `cache.has` followed by `cache.get` either since that's an extra call to Redis.